Git-commit: 8f8fb5ba80a857c800dc524917fabfb547c75309
From: Wang ShaoBo <bobo.shaobowang@huawei.com>
Date: Sat, 27 Feb 2021 18:35:31 +0800
Subject: [PATCH] openeuler_defconfig: Enable MPAM by default
Patch-mainline: Queued in openEuler repo, version 5.10.0-4.5.0
References: openEuler-21.03
Git-repo: https://gitee.com/openeuler/kernel.git

hulk inclusion
category: feature
feature: ARM MPAM support
bugzilla: 48265
CVE: NA

--------------------------------

Enable MPAM by default.

Signed-off-by: Wang ShaoBo <bobo.shaobowang@huawei.com>
Reviewed-by: Xie XiuQi <xiexiuqi@huawei.com>
Signed-off-by: Zheng Zengkai <zhengzengkai@huawei.com>
Signed-off-by: Kai Liu <kai.liu@suse.com>
---
 arch/arm64/configs/openeuler_defconfig | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index b5ceeaa4779c..a4dbaa42e49d 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -7097,5 +7097,8 @@ CONFIG_CC_HAS_SANCOV_TRACE_PC=y
 # CONFIG_MEMTEST is not set
 # end of Kernel Testing and Coverage
 # end of Kernel hacking
+CONFIG_MPAM=y
+CONFIG_RESCTRL=y
+CONFIG_ACPI_MPAM=y
 CONFIG_MEMIG_SCAN=m
 CONFIG_MEMIG_SWAP=m
-- 
2.31.1

